src/LocalizationWrapper.jsx

```diff
@@ -1,33 +1,36 @@
-import React, {Component, PropTypes} from 'react';
+import React, { Component, PropTypes } from 'react';
 import get from 'lodash.get';
 import defaultLocalizer from './util/localize-formatter';
 
 export default (ComposedComponent, messages = {}, customLocalizer) => {
   class ConfigureLocalization extends Component {
-    constructor(props) {
-      super(props);
+    static localize(key, values) {
+      const message = get(messages, key, null);
```

Comment:
It can be, or in some of our apps we have `'label.foo'` as the key, so `get` seemed safest to use for a default. You can always override this method when you use either this wrapper _or_ the `<Localization/>` component.